### PR TITLE
Frontend updates

### LINKED ISSUE #80: Front end updates
- [x] Change end of the first wave date for the map colour to the maximum date of Belgium
- [x] Change text reflecting this information in the Header (write maximum date), make sure text about simulation is accurate.
- [x] Left of the info panel with information from the "natural history"
- [x] Right of the infopanel with information from the simulation (dates of the simulation, total number of cases, % change wrt natural history of the counterfactual simulation (placeholder for now))
- [ ] ~Add restriction dates contrains to calendar~ (moved to its own issue)
- [x] Fix calendar format.
- [x] Fix calendar issue

@KFArnold if you have any other suggestions please just let them as a comment.

### PR DESCRIPTION
**Pull request by crangelsmith**
Fixes #80

@jemrobinson this PR has some frontend updates as described on #80, and is ready to be review. We can go through it together in the meeting today.

### CODE REVIEW

File: frontend/src/components/InfoPanel.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default class InfoPanel extends React.Component {
       task.getCountryRestrictionDates(this.props.isoCode),
     ]);
 
-    if ((restrictionsDates.length != 0) & (this.props.isoCode != null)) {
+    if ((restrictionsDates != null) & (this.props.isoCode != null)) {
```
Comment (jemrobinson) [inline frontend/src/components/InfoPanel.jsx:38]: We might also need to check that `restrictionsDates.first_restrictions_date` exists before trying to use it.

File: frontend/src/components/InfoPanel.jsx [hunk 2]
```diff
@@ -119,37 +119,50 @@ export default class InfoPanel extends React.Component {
...
-                      <Card.Title>Covid-19 Statistics:</Card.Title>
+                      <Card.Title>Statistics</Card.Title>
                       <Card.Text>
-                        {`Total Cases per Million: ${this.props.summedAvgCases
+                        {`Total Covid-19 Cases per Million: ${this.props.summedAvgCases
                           .toFixed(2)
                           .toString()} \n `}
                       </Card.Text>
-                      <Card.Text>{`Total Deaths per Million: XXX`}</Card.Text>
+                      <Card.Text>{`Total Covid-19Deaths per Million: XXX`}</Card.Text>
```
Comment (jemrobinson) [inline frontend/src/components/InfoPanel.jsx]:
```suggestion
                      <Card.Text>{`Total Covid-19 Deaths per Million: XXX`}</Card.Text>
```

File: frontend/src/components/InfoPanel.jsx (hunk 1 above)
Comment (jemrobinson) [reply frontend/src/components/InfoPanel.jsx:38]: Also note that `updateCounterfactual` is always `true` whenever we call this function. Do we need the variable?

**Review by jemrobinson — APPROVED** [review]

File: frontend/src/components/InfoPanel.jsx (hunk 1 above)
Comment (crangelsmith) [reply frontend/src/components/InfoPanel.jsx:38]: Suggestions implemented in fe8bbab and e15cc7f.

**Comment by crangelsmith** [comment]
Comments from today's meeting are now implemented. I'll merge this PR.

**Pull request merged**